script/translations.js

```diff
@@ -0,0 +1,18 @@
+window.translations = {
```

Comment:
Could we use a more unique sounding variable name? Just cause stuff on the window can get a little messy. Alternatively if you have any ideas for not putting this on the window! I thought briefly about either jambo building out a static js file for us that we could import directly, but that seemed like a lot of gymnastics just for this.

Reply:
what about `hitchhiker_theme_translations`? I'm not sure if there's another way of passing information between partial and static file at the moment. Jambo doesn't generate any static js files currently right? so that would need some changes to support that

Reply:
yeah that works! Just something a little more unique for when you need to `ctrl + f` it, since you can use window variables without specifying the `window.` prefix